src/BinSkim.Rules/PERules/BA2006.BuildWithSecureTools.cs

```diff
@@ -132,33 +141,43 @@ public override void AnalyzePortableExecutableAndPdb(BinaryAnalyzerContext conte
                         break;
                     }
 
-                    //case Language.MASM:
-                    //{
-                    //    minCompilerVersion =
-                    //        context.Policy.GetProperty(MinimumToolVersions)[nameof(Language.MASM)];
-                    //    break;
-                    //}
-
-                    //case Language.CVTRES:
-                    //{
-                    //    minCompilerVersion =
-                    //        context.Policy.GetProperty(MinimumToolVersions)[nameof(Language.CVTRES)];
-                    //    break;
-                    //}
-
-                    //case Language.CSharp:
-                    //{
-                    //    minCompilerVersion =
-                    //        context.Policy.GetProperty(MinimumToolVersions)[nameof(Language.CSharp)];
-                    //    break;
-                    //}
+                    /*
+                    TODO: MikeFan (1/6/2022)
+                    We need to take a step back and comprehensively review our compiler/language support.
+                    https://github.com/Microsoft/binskim/issues/114
+
+                    case Language.MASM:
+                    {
+                        minCompilerVersion =
+                            context.Policy.GetProperty(MinimumToolVersions)[nameof(Language.MASM)];
+                        break;
+                    }
+
+                    case Language.CVTRES:
+                    {
+                        minCompilerVersion =
+                            context.Policy.GetProperty(MinimumToolVersions)[nameof(Language.CVTRES)];
+                        break;
+                    }
+
+                    case Language.CSharp:
+                    {
+                        minCompilerVersion =
+                            context.Policy.GetProperty(MinimumToolVersions)[nameof(Language.CSharp)];
+                        break;
+                    }
 
+                    Language data is not always included if it is only compiled with SymTagCompiland without SymTagCompilandDetails
+                    https://docs.microsoft.com/en-us/visualstudio/debugger/debug-interface-access/compilanddetails?view=vs-2022
+                    Compiland information is split between symbols with a SymTagCompiland tag (low detail)
+                    and a SymTagCompilandDetails tag (high detail).
                     case Language.Unknown:
                     {
                         minCompilerVersion =
                             context.Policy.GetProperty(MinimumToolVersions)[nameof(Language.Unknown)];
                         break;
                     }
```

Comment:
This one wasn't commented out in previous iterations. Was this change intentional?